metricbeat/helper/prometheus/textparse.go

```diff
@@ -395,6 +396,10 @@ func summaryMetricName(name string, s float64, qv string, lbls string, t *int64,
 	return name, metric
 }
 
+/*
+histogramMetricName returns the metric name without the suffix and its histogram.
+OpenMetric suffixes: https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md#suffixes.
```

Comment:
This helper is used by Prometheus and Openmetrics module at the same time. Having a link only from OpenMetric spec might be confusing here. Could you either add one for Prometheus as well and explain that it covers both or explain it specifically if it covers both?